### DISCUSSION TITLE
Change Preheat/Cooldown menu position

### DISCUSSION THREAD

**Discussion by TitanwaCS** [Jyers/Marlin#1092] [Ideas]
### I think the preheat and cooldown menu should be moved upward 

**From this**
MOVE
DISABLE STEPPER
HOMING
MANUAL LEVELING
Z-OFFESET
PREHEAT
COOLDOWN
CHANGE FILAMENT

**To This**
MOVE
DISABLE STEPPER
HOMING
PREHEAT
COOLDOWN
MANUAL LEVELING
Z-OFFESET
CHANGE FILAMENT

Reason - It's easy and faster to reach preheat menu then scrolling down 
               Also manual levling and z-offset menu are used very less

**Comment by smooreace** [comment]
I tend to agree that preheat should be higher. Its potentially something
done prior to every print, and always done prior to a bed leveling

**Comment by TitanwaCS** [comment]
@Jyers Thoughts?

**Comment by Jyers** [comment]
Yeah, I think that's a great pattern. I'm also considering moving cooldown into preheat... That way you get the option to only cooldown the bed or hotend if you want that and it saves a menu item in prepare
